tokio/src/net/unix/listener.rs

```diff
@@ -104,7 +104,8 @@ impl UnixListener {
         poll_fn(|cx| self.poll_accept(cx)).await
     }
 
-    pub(crate) fn poll_accept(
+    #[doc(hidden)]
+    pub fn poll_accept(
```

Comment:
Could you add docs like `TcpListener::poll_accept`?: 

https://github.com/tokio-rs/tokio/blob/207320dbbbacb4e7c8c12f6e94f5e78eb0f055a5/tokio/src/net/tcp/listener.rs#L188-L192

(then, you could remove `#[doc(hidden)]`)